redash/handlers/data_sources.py

```diff
@@ -75,8 +75,10 @@ def post(self):
 
 class DataSourceSchemaAPI(BaseResource):
     def get(self, data_source_id):
+        refresh = request.args.get("refresh", "0", type=bool)
```

Comment:
I don't think this works the way you think it works. From what I've seen, if no refresh value is passed, it will return "0" (as string), which is a truth value... also, calling `bool` on any string value (including 0) will return True.